conf/application.conf

```diff
@@ -91,6 +91,7 @@ features {
   # if it is false, / will be the login page, if not logged in, and the dashboard otherwise
   isDemoInstance = false
   autoBrushReadyDatasets = []
+  taskReopenAllowedFrontend = 30000
```

Comment:
I assume this is in milliseconds? I'd change it to seconds (maybe add a `inSeconds` suffix to the var name).